src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseIntegrationServiceTest.java

```diff
@@ -336,16 +422,24 @@ public void textExportSubmissionsByParticipationIds_instructorNotInCourse_forbid
         request.postWithResponseBodyFile(path, getOptions(), HttpStatus.FORBIDDEN);
     }
 
-    public void textExportSubmissionsByStudentLogins() throws Exception {
+    public void testExportSubmissionsByStudentLogins() throws Exception {
+        File downloadedFile = exportSubmissionsByStudentLogins(HttpStatus.OK);
+        assertThat(downloadedFile).exists();
+        // TODO: unzip the files and add some checks
```

Comment:
What about this TODO? Can we remove it?

Reply:
This was already here, I just changed the code up a bit